inventory/any.cf

```diff
@@ -141,22 +141,43 @@ bundle agent cfe_autorun_inventory_memory
 # @brief Inventory the memory (Enterprise only)
 {
   vars:
-    enterprise.!aix::
-      # due to a Windows issue this is set to 0 there for now
-      "total" string => ifelse("windows", "0",
-                               $(mon.value_mem_total)),
-      meta => { "inventory", "attribute_name=Memory size (MB)" };
-
-    enterprise::
-      "free" string => ifelse("windows", "0",
-                              $(mon.value_mem_free)),
-      meta => { "report" };
-
+@if minimum_version(3.11)
+      # The `with` attribute is necessary for this to work in a single promise.
```

Comment:
Shouldn't there be an @else for this minimum_version guard? What happens in our 3.10 LTS for this? Shouldn't it return 0 for total and free as before?

Reply:
AFAIK we do not have an @else . The important thing is that the parser will complain on any version older than 3.11 if it sees the with the attribute in use.

reporting 0 is lame.

Reply:
Ok. No @else. @nickanderson I tested with 3.10 and 3.12 agents and this policy seems fine. I think the difference is not too important.